clients/admin-ui/src/features/config-wizard/ConfigWizardWalkthrough.tsx

```diff
@@ -52,6 +53,13 @@ const ConfigWizardWalkthrough = () => {
                 }
               />
             ) : null}
+            {step === 2 ? (
```

Comment:
I didn't think of this earlier when reviewing the OrganizationInfo PR, but I wonder if these steps should be their own routes/links as opposed to internal react state? When I was playing around on this branch, I realized I couldn't use the browser back button to go back a step (and actually there is no way to go back a step right now, I think? can't seem to click on the stepper). this leads to some sad UX since then you have to make another organization, but you can't actually since we are using the same `fides_key` (is there a reason for that?) More of a design/product question, just thought I'd note it here.

Reply:
Actually, that's a really great point and not something I considered - the designs don't appear to have any sort of "back" capability, and I wonder if this is intentional. I'm going to tag design here in this comment to get their input. Thanks for pointing it out!

Reply:
From Simon:
"Hey [@Laura](https://ethyca.slack.com/team/U038HU329B4) apologies if this wasn’t clear. My thinking is that stepper itself serves as the navigation to go back if required. So at any point in the setup the user has two choices :
Cancel setup and return to the wizard home page
Use the stepper to navigate back to previous steps
Let me know if you need any further details on this or any other states mocked up and I’ll get to them today for you."

With this info, I'm going to create a follow-up ticket/PR to add that functionality to the Wizard Stepper, if you're okay with that. I'll work on it next.